Google.PowerShell/ReleaseFiles/GoogleCloud.psm1

```diff
@@ -50,10 +62,10 @@ function Install-GCloudSdk {
     }
 
     Write-Host "Google Cloud SDK is not found in PATH. The SDK is required to run the module."
-    $noPrompt = $env:GCLOUD_SDK_INSTALLATION_NO_PROMPT -eq $true -or $args -match "-?quiet"
```

Comment:
Is this a merge error, or did you not like this feature?

Reply:
It's a merge error, I'll add it back.